client/signup/controller.js

```diff
@@ -71,6 +72,12 @@ const gutenbergRedirect = function ( flowName, locale ) {
 	window.location.replace( url.toString() );
 };
 
+const getSideIdOrSlug = ( signupDependencies, query ) => {
```

Comment:
oops, typo in the method name
```suggestion
const getSiteIdOrSlug = ( signupDependencies, query ) => {
```